ORA.Core/Compression/ZipLibCompressor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+using ORA.API.Compression;
+using ICSharpCode.SharpZipLib.Zip.Compression;
+
+namespace ORA.Core.Compression
+{
+    public class ZipLibCompressor : ICompressor
+    {
+
+        private static int n;
```

Comment:
Not working if you compress two data and you want to decrompress the first compressed data.